docs/monitoring-usage.rst

```diff
@@ -278,3 +278,91 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation on `Custom Metrics`_ for more information.
+
+To write a data point to a custom metric, you must provide an instances of
+:class:`~gcloud.monitoring.metric.Metric` created with a metric type as well as values for
+the labels associated with that metric type. The metric type can either
+be created using the :class:`~gcloud.monitoring.metric.MetricDescriptor` class or
+auto-created.  For more information on creating metric descriptors, see the `Metric Descriptors`_
+section. For information on metric type auto-creation, see `Auto-creation of custom metrics`_.
+
```

Comment:
Hmm, I don't see how the confusion remains. Your suggested replacement sentences seem very similar to what is written minus the link to the class and previous reference docs. If I say "You must have created the metric descriptor earlier" it raises the question, how do I create it, which is why I think those links are helpful. Auto-creation is currently linked to.